src/Microsoft.TestPlatform.Client/TestPlatform.cs

```diff
@@ -76,6 +82,8 @@ public ITestRunRequest CreateTestRunRequest(TestRunCriteria testRunCriteria)
                 throw new ArgumentNullException("testRunCriteria");
             }
 
+            UpdateTestAdapterPaths(testRunCriteria.TestRunSettings);
+
```

Comment:
This caused issue in Tpv1, when in VS IDE scenario data collectors were enabled, & runsettings were used to provide adapter path, it resulted in DataCollection(fakes, CodeCoverage) not working.

The workaround we used was to not update them here, but instead like we process TestAdapterPath in CLI, similarly process it, & apply it on ITestPlatform when provided via runsettings